layers/+distributions/spacemacs-base/packages.el

```diff
@@ -14,6 +14,7 @@
         (abbrev :location built-in)
         ace-window
         (bookmark :location built-in)
+        (centered-buffer-mode :location local)
```

Comment:
Write a README, document all the stuff. Then if I become a successful project owner,  some internet activists gonna find my repo and start  forcing their CoC on me with ultimatums and no negotiations. NOoo! :scream: